Source/Evolve/WarpXEvolve.cpp

```diff
@@ -523,6 +517,14 @@ void WarpX::HandleParticlesAtBoundaries (int step, amrex::Real cur_time, int num
         }
     }
 
+    // interact the particles with EB walls (if present)
+#ifdef AMREX_USE_EB
+    mypc->ScrapeParticles(amrex::GetVecOfConstPtrs(m_distance_to_eb));
```

Comment:
While these changes are being made, this routine could be renamed to be more explicit, something like `ScrapeParticlesAtEB`.

Reply:
Sounds good!